Skips test that has been skipped by devs while being investigated
See https://github.com/scikit-bio/scikit-bio/issues/2372#issuecomment-3881284157
Author: Pedro Santos Neves - CIT-HPC UGroningen <p.m.santos.neves@rug.nl>
--- skbio/stats/composition/tests/test_dirmult.py  		2025-10-30 18:53:25.000000000 +0100
+++ skbio/stats/composition/tests/test_dirmult.py	     	2026-02-11 14:25:37.022619973 +0100
@@ -322,16 +322,20 @@
         #     0.247, 0.445, 0.465, 0.766, 0.190, 0.695, 0.257, 0.415]))
 
         # add a random slope for Covar2
-        res = dirmult_lme(
-            table=self.table, metadata=self.metadata, formula="Covar2 + Covar3",
-            grouping="Covar1", draws=1, seed=0, p_adjust="sidak",
-            re_formula="1 + Covar2", fit_method="bfgs")
-        self.assertIsInstance(res, pd.DataFrame)
+        # Patch: temporarily remove this test
+        # see https://github.com/scikit-bio/scikit-bio/issues/2372,
+        # https://github.com/scikit-bio/scikit-bio/pull/2378 and
+        # https://github.com/scikit-bio/scikit-bio/issues/2379
+        #res = dirmult_lme(
+        #    table=self.table, metadata=self.metadata, formula="Covar2 + Covar3",
+        #    grouping="Covar1", draws=1, seed=0, p_adjust="sidak",
+        #    re_formula="1 + Covar2", fit_method="bfgs")
+        #self.assertIsInstance(res, pd.DataFrame)
         # NOTE: This test function is prone to generate different results on different
         # platforms. If the following test is to be executed, be prepared that lower
         # precision is needed to get the test pass.
-        npt.assert_array_equal(res["qvalue"].round(3), np.array([
-            0.274, 0.463, 0.645, 0.766, 0.251, 0.703, 0.400, 0.415]))
+        # npt.assert_array_equal(res["qvalue"].round(3), np.array([
+        #    0.274, 0.463, 0.645, 0.766, 0.251, 0.703, 0.400, 0.415]))
 
     def test_dirmult_lme_vc_formula(self):
         res = dirmult_lme(
